src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -71,17 +71,7 @@ namespace winrt::TerminalApp::implementation
         for (const auto& nameAndCmd : commands)
         {
             const auto& command = nameAndCmd.Value();
-            // If there's a keybinding that's bound to exactly this command,
-            // then get the string for that keychord and display it as a
-            // part of the command in the UI. Each Command's KeyChordText is
-            // unset by default, so we don't need to worry about clearing it
-            // if there isn't a key associated with it.
-            auto keyChord{ settings.KeyMap().GetKeyBindingForActionWithArgs(command.Action()) };
 
-            if (keyChord)
-            {
-                command.KeyChordText(KeyChordSerialization::ToString(keyChord));
```

Comment:
Surprisingly, we still need something like this to support the following scenario:
```
        {
            "name": "Test...",
            "commands": [
                { "name": "hello", "command": "copy" },
                { "name": "custom", "command": "openSettings", "keys": "ctrl+q" }
            ]
        },
```

In this situation, "copy" still gets the "ctrl+c" key binding, but "openSettings" does not.

A fix will be included in the upcoming commit.

Reply:
For the record, the current implementation displays "hello" without a kbd, but "custom" with one. Exactly backwards. And `ctrl+q` does _not_ actually execute the action.

Reply:
This is now fixed. As a part of `ActionMap`, it would be nice if we didn't have to do this. But I'm keeping an eye out to make sure this works as intended in the next PR.

For now, this is fine. `ActionMap` will probably get some tests to make sure we get the behavior we want.